src/network/sniffer.rs

```diff
@@ -157,6 +166,12 @@ impl Sniffer {
         let from = SocketAddr::new(ip_packet.get_source().into(), source_port);
         let to = SocketAddr::new(ip_packet.get_destination().into(), destination_port);
 
+        if !show_dns {
+            if from.port() == 53 {
+                return None;
+            }
+        }
+
```

Comment:
I think it would be more accurate to do this after creating the connection... something like:
```rust
         if !show_dns {                                                                                                                                                                                                                     
             if connection.remote_socket.port == 53 {
                 return None;
             }
         }
```

Reply:
Hi @imsnif! Thank you, yeah, sure, i just was going to add a tests for this. This PR should be work in progress yet.

Reply:
@imsnif Good afternoon, could you please check my pr?